config/samples/udproute-noreach/kustomization.yaml

```
@@ -0,0 +1,17 @@
+
+resources: 
+- ../udproute
+patchesStrategicMerge:
+- |-
+  apiVersion: apps/v1
+  kind: Deployment
+  metadata:
+    name: blixt-udproute-sample
+  spec:
+    template:
+      spec:
+        containers:
+        - name: server
+          command:
+            - ./udp-test-server
+            - --dry-run
```

Should we add a comment on this as a breadcrumb to the user as the entire point of this patch is to apply this?

Maybe something like:

```suggestion
            # --dry-run disables UDP listeners in order to test failures to send data, and trigger ICMP port failure responses from the kernel
            - --dry-run
```

---

Yep good Call I will add